utils/oscap.8

```diff
@@ -310,7 +310,7 @@ To use the ability to include additional information from SCE in XCCDF result fi
 .TP
 .B \fBfix\fR  [\fIoptions\fR] xccdf-file
 .RS
-Generate a script that shall bring the system to a state of compliance with given XCCDF Benchmark.
+Generate a script that shall bring the system to a state of compliance with given XCCDF Benchmark. There are 3 possibilities when generating fixes: Result-oriented fixes (--result-id), Profile-oriented fixes (--profile) or Default Profile-oriented fixes (no option provided to oscap). Result-oriented takes precedences over Profile-oriented, so if you provide --result-id oscap will ignore any profile provided.
```

Comment:
I added 2 lines defining what are Profile/Result-oriented fixes, not sure if they are clear, comments are welcome! :smile: